hack/verify-flags-underscore.py

```diff
@@ -184,88 +116,26 @@ def get_flags(rootdir, files):
                 new_excluded_flags.add(flag)
             if not "-" in flag:
                 continue
-            if flag not in flags:
-                new_flags.add(flag)
+            flags.add(flag)
     if len(new_excluded_flags) != 0:
         print("Found a flag declared with an _ but which is not explicitly listed as a valid flag name in hack/verify-flags/excluded-flags.txt")
         print("Are you certain this flag should not have been declared with an - instead?")
         l = list(new_excluded_flags)
         l.sort()
         print("%s" % "\n".join(l))
         sys.exit(1)
-    if len(new_flags) != 0:
-        print("Found flags with character - in golang files not in the list of known flags. Please add these to hack/verify-flags/known-flags.txt")
-        l = list(new_flags)
-        l.sort()
-        print("%s" % "\n".join(l))
-        sys.exit(1)
     return list(flags)
 
-def flags_to_re(flags):
-    """turn the list of all flags we found into a regex find both - and _ versions"""
-    dashRE = re.compile('[-_]')
-    flagREs = []
-    for flag in flags:
-        # turn all flag names into regexs which will find both types
-        newre = dashRE.sub('[-_]', flag)
-        # only match if there is not a leading or trailing alphanumeric character
-        flagREs.append("[^\w${]" + newre + "[^\w]")
-    # turn that list of regex strings into a single large RE
-    flagRE = "|".join(flagREs)
-    flagRE = re.compile(flagRE)
-    return flagRE
-
-def load_exceptions(rootdir):
-    exceptions = set()
-    if args.skip_exceptions:
-        return exceptions
-    exception_filename = os.path.join(rootdir, "hack/verify-flags/exceptions.txt")
-    exception_file = open(exception_filename, 'r')
-    for exception in exception_file.read().splitlines():
-        out = exception.split(":", 1)
-        if len(out) != 2:
-            print("Invalid line in exceptions file: %s" % exception)
-            continue
-        filename = out[0]
-        line = out[1]
-        exceptions.add((filename, line))
-    return exceptions
-
 def main():
     rootdir = os.path.dirname(__file__) + "/../"
     rootdir = os.path.abspath(rootdir)
 
-    exceptions = load_exceptions(rootdir)
-
     if len(args.filenames) > 0:
         files = args.filenames
     else:
         files = get_all_files(rootdir)
-    files = normalize_files(rootdir, files)
-
-    check_known_flags(rootdir)
-
-    flags = get_flags(rootdir, files)
-    flagRE = flags_to_re(flags)
-
-    bad_lines = []
-    # walk all the file looking for any flag that was declared and now has an _
```

Comment:
I'm not sure why we had this check (I'm guessing it's what caused the slowness), so it seems fine to me to remove this.